services/gitdiff/gitdiff_test.go

```diff
@@ -643,9 +642,9 @@ func TestGetDiffRangeWithWhitespaceBehavior(t *testing.T) {
 				MaxFiles:           setting.Git.MaxGitDiffFiles,
 				WhitespaceBehavior: behavior,
 			})
-		assert.NoError(t, err, fmt.Sprintf("Error when diff with %s", behavior))
+		assert.NoError(t, err, "Error when diff with %s", behavior)
 		for _, f := range diffs.Files {
-			assert.True(t, len(f.Sections) > 0, fmt.Sprintf("%s should have sections", f.Name))
+			assert.Positive(t, f.Sections, "%s should have sections", f.Name)
```

Comment:
Like this?

```suggestion
			assert.NotEmpty(t, f.Sections, "%s should have sections", f.Name)
```

(I am not sure whether "Positive" works for a slice)